pentagon/migration/migrations/migration_2_7_1.py

```diff
@@ -0,0 +1,112 @@
+from copy import deepcopy
+
+
+from pentagon import migration
+from pentagon.migration import *
+import yaml
+
+readme = """
+
+# Migration 2.7.1 -> 2.7.2
+
+## This migration:
+- adds kubelet flags that were missing in the last migration to take advantage of the audit policy
+
+
+## Risks:
+- this requires you to roll the cluster
+
+## Follow up tasks:
+- roll the cluster
+
+"""
+
+# Magic to make block formatting in yaml.dump work as expected
+
+
+class folded_unicode(unicode):
+    pass
+
+
+class literal_unicode(unicode):
+    pass
+
+
+def folded_unicode_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+
+
+def literal_unicode_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+
+yaml.add_representer(folded_unicode, folded_unicode_representer)
+yaml.add_representer(literal_unicode, literal_unicode_representer)
+# https://stackoverflow.com/questions/6432605/any-yaml-libraries-in-python-that-support-dumping-of-long-strings-as-block-liter
+
+audit_settings = {
+    'auditLogPath': '/var/log/kube-apiserver-audit.log',
+    'auditLogMaxAge': 10,
+    'auditLogMaxBackups': 1,
+    'auditLogMaxSize': 100,
+    'auditPolicyFile': '/srv/kubernetes/audit.yaml'
+}
+
+
+class Migration(migration.Migration):
+    _starting_version = '2.7.1'
+    _ending_version = '2.7.2'
+
+    _readme_string = readme
+
+    def run(self):
+
+        old_nodes_file_name = "nodes.yml"
```

Comment:
appears to be defined but not used? Is this used anywhere?

Reply:
Nope, copy pasta leftovers